automation/README.md

```diff
@@ -3,17 +3,9 @@ The automation env is serving the tests of Nmstate.
 It may be used both locally and through CI.
 
 ## Components
-- Dockerfile: Defines a Centos 7 based container image which includes systemd,
-  NetworkManager and other basic build tools (e.g. tox).
-
-  The image can be found at:
-  https://hub.docker.com/r/nmstate/centos7-nmstate-dev/
-
-- Dockerfile.fedora: Defines a Fedora based container image for Nmstate test
```

Comment:
Why do you remove this?

Reply:
The container specifications are in https://github.com/nmstate/nmstate/tree/master/packaging and the file is called differently there.